chasm/lib/callback/executors_test.go

```diff
@@ -461,9 +456,10 @@ func TestExecuteInvocationTaskChasm_Outcomes(t *testing.T) {
 				require.NoError(t, err)
 				return comp
 			}(),
-			headerValue:          encodedRef,
-			expectsInternalError: true,
-			assertOutcome: func(t *testing.T, cb *Callback) {
+			headerValue: encodedRef,
+			assertOutcome: func(t *testing.T, cb *Callback, err error) {
+				require.Error(t, err)
+				require.Contains(t, err.Error(), "internal error, reference-id:")
```

Comment:
This can be shortened to `require.ErrorContains` but ideally you would also check the error type if a concrete type is relevant and available (didn't check).